packages/messenger-widget/src/components/RightHeader/NormalView.tsx

```diff
@@ -0,0 +1,111 @@
+import './RightHeader.css';
+import { ConnectButton } from '@rainbow-me/rainbowkit';
+import { useContext, useEffect, useState } from 'react';
+import humanIcon from '../../assets/images/human.svg';
+import menuIcon from '../../assets/images/menu.svg';
+import { AuthContext } from '../../context/AuthContext';
+import { useMainnetProvider } from '../../hooks/mainnetprovider/useMainnetProvider';
+import { HideFunctionProps } from '../../interfaces/props';
+import { GlobalContext } from '../../utils/context-utils';
+import { getAvatarProfilePic } from '../../utils/ens-utils';
+import {
+    RightViewSelected,
+    UiViewStateType,
+} from '../../utils/enum-type-utils';
+import { ConversationContext } from '../../context/ConversationContext';
+
+export function NormalView(props: HideFunctionProps) {
+    // fetches context storage
+    const { state, dispatch } = useContext(GlobalContext);
+    const { account, displayName } = useContext(AuthContext);
+    const { setSelectedContactName } = useContext(ConversationContext);
+
+    const mainnetProvider = useMainnetProvider();
+
+    // state to store profile pic of signed in user
+    const [profilePic, setProfilePic] = useState<string>('');
+
+    // fetches profile pic of contact for mobile screens
+    // fetched profile pic of signed in user for desktop
+    const fetchAndSetProfilePic = async () => {
+        setProfilePic(
+            await getAvatarProfilePic(
+                mainnetProvider,
+                account?.ensName as string,
+            ),
+        );
+    };
+
+    // method to set profile page and set contact
+    const updateView = () => {
+        if (props.showContacts) {
+            let profileActive: RightViewSelected =
```

Comment:
Please change to 
const profileActive = state.uiView.selectedRightView ===RightViewSelected.Profile
                    ? RightViewSelected.Default
                    : RightViewSelected.Profile;